src/oca_github_bot/tasks/merge_bot.py

```diff
@@ -174,10 +173,7 @@ def _prepare_merge_bot_branch(
     elif merge_strategy == MergeStrategy.rebase_autosquash:
         # rebase the pr branch onto the target branch
         _git_call(["git", "checkout", pr_branch])
-        _git_call(
-            ["git", "rebase", "--autosquash", "-i", target_branch],
-            env=dict(os.environ, GIT_SEQUENCE_EDITOR="true"),
-        )
+        _git_call(["git", "rebase", "--autosquash", "-i", target_branch])
```

Comment:
@SimoRubi thanks for tackling this. Actually the current version of this part of the code should be correct.

The thing is, this `rebase_autosquash` merge strategy is never triggered by the current code base.
This is because I changed the default merge strategy from rebase to merge in #70, because is a more familiar default, it preserve commit signatures, etc.

rebase with autosquash should now be asked explicitly in the merge command.

If you think a rebase merge strategy (and autosquash) are desirable features, I suggest you open an issue to specify now "ocabot merge" options to enable that.

Sorry for the confusion.

Reply:
I've updated the blog post to mention the new default merge strategy.